src/widgetBases.d.ts

```diff
@@ -179,145 +172,62 @@ export interface ContainerListMixin<C extends Renderable> {
 }
 
 /**
- * The *final* type for ContainerListWidget
+ * The *final* type for ContainerWidget
  */
-export type ContainerListWidget<C extends Renderable, S extends ContainerListWidgetState> = Widget<S> & ContainerListMixin<C>;
+export type ContainerWidget<C extends Renderable, S extends ContainerWidgetState> = Widget<S> & ContainerWidgetMixin<C>;
 
-export interface ContainerListWidgetOptions<C extends Renderable, S extends ContainerListWidgetState> extends WidgetOptions<S> {
+export interface ContainerWidgetOptions<C extends Renderable, S extends ContainerWidgetState> extends WidgetOptions<S> {
 	/**
 	 * A list of children to be created and added to the container at creation time
 	 */
 	createChildren?: [Factory<C, WidgetOptions<WidgetState>>, WidgetOptions<WidgetState> ]
```

Comment:
do we need `[Factory<C, WidgetOptions<WidgetState>>, WidgetOptions<WidgetState> ]` and `[Factory<C, WidgetOptions<WidgetState>>]` given that `CreateWidgetList` is typed as `([Factory<W, O>, O ] | [Factory<W, O>])`?